src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -160,7 +160,39 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.reloadPage();
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditSuccess.html");
 
-
+        ______TS("test two 'change' links' functionality to expand uncommon settings panels");
+        
+        // test uncommon settings for 'send emails'
+        feedbackEditPage.clickEditUncommonSettingsSendEmailsButton();
+        feedbackEditPage.verifyEditSessionBoxIsEnabled();
+        feedbackEditPage.toggleClosingSessionEmailReminderCheckbox();
+        feedbackEditPage.clickSaveSessionButton();
+        feedbackEditPage.reloadPage();
```

Comment:
It's a "hack" solution but valid. :-P
a4dd038c31e4b5a31ea3058be475ccc0baeeaca5
I add test codes to checking status message
``` java
feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_EDITED);
```
between the submission and verification in the test cases to actually give time for the data store (the verification of status message (string comparison) actually takes time.)